### PR TITLE
feat: Create mentorship overview API

### LINKED ISSUE #28: API: Mentorship Overview Page
Image

### PR DESCRIPTION
**Pull request by dricazenck**
Closes #28 

## Page Details
Image

## API Doc
image

### CODE REVIEW

**Review by nverbic — APPROVED** [review]
All looks good to me. It can be merged.

File: src/main/resources/mentorshipPage.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (dricazenck) [inline src/main/resources/mentorshipPage.json:1]: @stepsen89 here is the api contract, for you to check.

File: README.md [hunk 2]
```diff
@@ -22,46 +22,55 @@
 
 * Access application on http://localhost:8080/api/cms/v1/team
 
-
-
 ## JAVA 21
-This project uses Java 21, you can run in 21.0.2 or 21.0.3. If you have installed a different version on your machine and don't want to remove it, you can use **SDKMAN** development tool.
-It will allow you to switch based on the Java version you want to use. 
-Here is the [link](https://sdkman.io).
```
Comment (nverbic) [inline README.md:30]: Add note to install lombok plugin

Comment (dricazenck) [reply README.md:30]: fixed in the next pr.

**Pull request merged**